sql/sql_class.cc

```diff
@@ -1656,6 +1657,7 @@ void THD::reset_for_reuse()
 #endif
 #ifdef WITH_WSREP
   wsrep_free_status(this);
+  wsrep_cs().reset_error();
```

Comment:
I wonder do we need to reset wsrep_aborter and wsrep__abort_by_kill here also or at least add assertions that they are not set.